src/js/themes/base.js

```diff
@@ -281,6 +281,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
           color: 'focus',
           size: '2px',
         },
+        allowAll: undefined,
```

Comment:
Pausing on the name of this property. From a Grommet convention, typically we haven't used verbs, instead use a noun / specific property.

How were we thinking of describing this in the documentation? Something like?:
```
If true, applies both outline and shadow styles to create a compound focus ring.
```
OR,
```
If true, applies both outline and shadow styles to create a two-color focus ring.
```

A couple naming options:
- `twoColor`
- `compound`

"Two-color" seemed to be the most common terminology used in search results.

Reply:
Good feedback, let's use the term `twoColor` since it speaks to the technique this is seeking to enable.